src/main/java/teammates/common/datatransfer/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -383,13 +383,13 @@ private String getDisplayableRecipientName(boolean isHiddenRecipient,
         
         // Replace current user name with "You"
         if (!isHiddenRecipient && isRecipientCurrentUser && hasAtLeastTwoResponses){
-            return isRecipientTypeStudent? "You" : "Your Team (" + currentUserTeam + ")";
+            return isRecipientTypeStudent? "You" : "Your Team (" + Sanitizer.sanitizeForHtml(currentUserTeam) + ")";
```

Comment:
We shouldn't sanitize inside this function. Ideally it should be right before injecting into HTML.